flaky/_flaky_plugin.py

```diff
@@ -94,6 +104,32 @@ def _handle_test_error_or_failure(self, test, err):
             self._log_test_failure(name, err, message)
             return False
 
+    def _should_rerun_test(self, test, name, err):
+        """
+        Whether or not a test should be rerun.
+        This is a pass-through to the test's rerun filter.
+
+        :param test:
+            The test that has raised an error
+        :type test:
+            :class:`nose.case.Test` or :class:`Function`
+        :param name:
+            The test name
+        :type name:
+            `unicode`
+        :param err:
+            Information about the test failure (from sys.exc_info())
+        :type err:
+            `tuple` of `class`, :class:`Exception`, `traceback`
+        :return:
+            True, if the test will be rerun;
+            False, if the test runner should handle it.
+        :rtype:
+            `bool`
+        """
+        rerun_filter = self._get_flaky_attribute(test, FlakyNames.RERUN_FILTER)
+        return rerun_filter(err, name, test, self)
```

Comment:
Does the test code below verify that the code here is passing the correct params. All the 'test' rerun_filter functions ignore the parameters so I'm not sure?